hoot-core/src/main/cpp/hoot/core/util/LogGeneric.cpp

```diff
@@ -111,18 +111,32 @@ bool Log::notFiltered(const string& prettyFunction)
     return true;
   }
 
-  // split arguments from function call name
-  QStringList nameParts = QString::fromStdString(prettyFunction).split("(");
-  if (nameParts.length() < 1) return true;
+  const QString prettyFunctionQt = QString::fromStdString(prettyFunction);
```

Comment:
Also see related changes in `LogJs`. The log class filtering wasn't working for logging occurring in generic conflate scripts, so fixed it.